website/content/api-docs/secret/pki.mdx

```diff
@@ -1186,6 +1186,16 @@ corresponding full CRL when it has been regenerated; otherwise, some serial
 numbers may not appear in the local copy of the full CRL if the remote
 complete and delta CRLs has been regenerated.
 
+Endpoints with source `local` only include cluster-local revocations. When
```

Comment:
nit: use `Local` and `Unified` to match table values?

Reply:
This is how we did it for `delta` vs `complete`, should I make them all consistent?

Reply:
ah then nvm.